synapse/lib/ast.py

```diff
@@ -1284,7 +1284,8 @@ class PropPivotOut(PivotOper):
                 fname = prop.type.arraytype.name
                 if runt.snap.model.forms.get(fname) is None:
                     if not warned:
-                        await runt.snap.warn(f'The source property "{name}" array type "{fname}" is not a form. Cannot pivot.')
+                        await runt.snap.warn(
```

Comment:
can we form the mesg a line above and avoid the multiline function call?

Reply:
ok